tmt/base.py

```diff
@@ -1106,6 +1106,8 @@ def environment(self) -> EnvironmentType:
             combined.update(self.my_run.environment)
             # Include path to the plan data directory
             combined["TMT_PLAN_DATA"] = self.data_directory
+            # And tree path
+            combined["TMT_TREE"] = self.worktree
```

Comment:
Hm, apparently according to the tests, `self.worktree` may not be defined, this edge case should somehow be handled

Reply:
Nice, good to have the test coverage. Fixed in newest commit.

Reply:
Defined or `None`? IIUIC, the attribute exists always, it's a `@property` of the `Common` class, so `AttributeError` shouldn't be a thing. But it may be `None`, therefore something like `if self.workdir` would be a more fitting check.

Reply:
@happz Previous commit has failing testsuite (in /tests/plan/import/basic ) with 
` AttributeError: 'Plan' object has no attribute 'worktree' `

Reply:
But good point, will add check to to skip if None

Reply:
Oh, sorry, I read it as "work**dir**" :/

Right, `worktree` may be undefined, it's not declared on class level, it's not created by `__init__()`, so, yeah, it may be missing. I'm not sure it's the best approach, because it's hard to catch by static analysis, as proven by the tests... Maybe consider declaring it as `Optional[str]` or something like that - it would probably spawn some amount of mypy complaints, but at least it'd be obvious where the field is used (and expected to exist...).

Reply:
Changed. Caused just one mypy complain.